scripts/extract_js_texts.js

```diff
@@ -3,7 +3,6 @@
 /* eslint-disable no-console */
 const BabelParser = require('@babel/parser');
 const color       = require('cli-color');
-const emphasize   = require('emphasize');
 const estraverse  = require('estraverse');
 const fs          = require('fs');
 const Path        = require('path');
```

Comment:
if we have import here maybe we can change all these requires here to import

Reply:
The following error doesn't let me replace 'require's with 'import's in this module:
>"Cannot use import declarations in modules that export using CommonJS (module.exports = 'foo' or exports.bar = 'hi')eslint(import/no-import-module-exports)"

I had to dynamically import only `emphasize` because only for this package I got the following error:
>"Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/maryiahryshchuk/Projects/smarttrader-binary-static/node_modules/emphasize/index.js from /Users/maryiahryshchuk/Projects/smarttrader-binary-static/scripts/extract_js_texts.js not supported.
Instead change the require of index.js in /Users/maryiahryshchuk/Projects/smarttrader-binary-static/scripts/extract_js_texts.js to a dynamic import() which is available in all CommonJS modules."

Reply:
You can change the last 3 lines to module export, then it will works with import i guess

```
// export parse function in function declaration
exports.parse          = parse;
// export function getTexts(app_name = this_app_name) {};
exports.getTexts       = (app_name = this_app_name) => source_strings[app_name];
// export function getErrorsCount(app_name = this_app_name) {};
exports.getErrorsCount = (app_name = this_app_name) => invalid_list[app_name].length;
```

Reply:
this doesn't work unfortunately. Also, as i mentioned [here](https://github.com/deriv-com/smarttrader/pull/617#discussion_r1533480060) :

An attempt to use 'import's and 'export default' causes another error:

>"Cannot use import statement outside a module"

because CommonJS is used in this project, not ESM.